.github/workflows/linter.yml

```
@@ -0,0 +1,20 @@
+name: ES Lint
+
+on:
+  push:
+    branches: [linter]
+  pull_request:
+    branches: [main]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Install dependencies
+        run: npm init @eslint/config
+      - name: Install typescript and eslint
+        run: npm install --save-dev @typescript-eslint/parser @typescript-eslint/eslint-plugin eslint typescript
+      - name: Lint it
+        run: npx eslint .
```

```suggestion
  lint:
    runs-on: ubuntu-latest
    name: ES Lint
    steps:
      - uses: actions/checkout@v3
      - uses: actions/setup-node@v3
      - run: npm install
      - name: Lint it
        uses: icrawl/action-eslint@v1
      - run: npx eslint .
```
You can try using a modified command instead of ```npx eslint .```
```eslint --ignore-path .gitignore . --ext ts --ext tsx --ext js --ext jsx```